packages/fuel-gauge/src/bytes.test.ts

```diff
@@ -133,16 +153,24 @@ describe('Bytes Tests', () => {
   });
 
   it('should test bytes input [script-bytes]', async () => {
-    const wallet = await setup();
-    type MainArgs = [number, Wrapper];
-    const scriptInstance = getScript<MainArgs, void>('script-bytes', wallet);
+    using launched = await launchTestNode({
+      walletsConfig: {
+        amountPerCoin: 500_000,
+      },
+    });
```

Comment:
```suggestion
    using launched = await launchTestNode();
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e04b